README.md

```diff
@@ -27,6 +27,8 @@ only), 1.9.3, 2.0.0 and 2.1.0.
  * EL 5
  * EL 6
  * EL 7
+ * FreeBSD 10
+ * FreeBSD 11
```

Comment:
Yep, FreeBSD-10.3-RELEASE  amd64 and FreeBSD 11-APLHA4 (not releases yet). Within the major version is no change, so I have a 10 and 11, instead of 10.0, 10.1, 10.2 ...